src/features/people/PeopleProfile/index.js

```diff
@@ -46,7 +46,7 @@ export const PeopleProfile = () => {
           />
         ))}
       </CastCrewSection>
-      <h2>Movies-Crew ({castCrew.crew.length})</h2>
+      <h2>Movies-Crew ({castCrew.crew?.length})</h2>
```

Comment:
Dorze, że udało się to rozwiązać. Dobra robota.

Reply:
Dzięki!